assets/icons/footer/discord_footer.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="23" height="23" viewBox="0 0 23 23" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
suggestion(blocking): use existing icons folder for svgs - applies for all icons from folder